[🐛 Bug]: Unit tests failing (see tests from PR #6296)

**Describe the bug**
Hi,

Three unit tests are currently failing due to calling a NoneType object and importing a non-existing function.

Logs:

`FAILED tests/test_incidents.py::test_incident_dto_is_visible_to_db - AttributeError: 'NoneType' object has no attribute 'order'
FAILED tests/test_keep_provider_notify_alert_none.py::test_notify_alert_with_none_alert_results_no_crash - AttributeError: <module 'keep.providers.keep_provider.keep_provider' from '/home/runner/work/keep/keep/keep/providers/keep_provider/keep_provider.py'> does not have the attribute 'Pusher'
FAILED tests/test_keep_provider_notify_alert_none.py::test_notify_alert_with_none_alert_results_and_alert_param - AttributeError: <module 'keep.providers.keep_provider.keep_provider' from '/home/runner/work/keep/keep/keep/providers/keep_provider/keep_provider.py'> does not have the attribute 'Pusher'`

**To Reproduce**
Steps to reproduce the behavior:

1. Submit a PR that runs the unit tests
2. Wait for the tests to complete.
3. See error

**Expected behavior**
All unit tests succeeding if the code is OK.

**Screenshots**
See unit tests from PR #6296

**Additional context**
Add any other context about the problem here.

--- a/tests/test_incidents.py
+++ b/tests/test_incidents.py
@@ -2037,11 +2037,14 @@ def test_incident_dto_is_visible_from_db(db_session, create_alert):
 def test_incident_dto_is_visible_to_db():
     """Test that is_visible is correctly mapped in IncidentDto.to_db_incident()."""
     from keep.api.models.incident import IncidentDto
+    from keep.api.models.db.incident import IncidentSeverity, IncidentStatus
     import datetime
 
     dto = IncidentDto(
         id=uuid4(),
         user_generated_name="Test",
+        severity=IncidentSeverity.INFO.order,
+        status=IncidentStatus.FIRING.value,
         is_predicted=False,
         is_candidate=False,
         is_visible=False,

--- a/tests/test_incidents.py
+++ b/tests/test_incidents.py
@@ -2043,7 +2043,7 @@ def test_incident_dto_is_visible_to_db():
     dto = IncidentDto(
         id=uuid4(),
         user_generated_name="Test",
-        severity=IncidentSeverity.INFO.order,
+        severity=IncidentSeverity.INFO,
         status=IncidentStatus.FIRING.value,
         is_predicted=False,
         is_candidate=False,

--- a/tests/test_keep_provider_notify_alert_none.py
+++ b/tests/test_keep_provider_notify_alert_none.py
@@ -51,8 +51,8 @@ def test_notify_alert_with_none_alert_results_no_crash():
         provider.io_handler = MagicMock()
         provider.io_handler.render_context.return_value = {}
 
-        # Mock pusher and alert DAO to avoid DB dependency
-        with patch("keep.providers.keep_provider.keep_provider.Pusher"), \
+        # Mock process_event and alert DAO to avoid DB dependency
+        with patch("keep.providers.keep_provider.keep_provider.process_event"), \
              patch.object(provider, "logger"):
             # Before the fix, this would raise:
             # TypeError: object of type 'NoneType' has no len()

--- a/tests/test_keep_provider_notify_alert_none.py
+++ b/tests/test_keep_provider_notify_alert_none.py
@@ -95,7 +95,7 @@ def test_notify_alert_with_none_alert_results_and_alert_param():
         provider.io_handler = MagicMock()
         provider.io_handler.render_context.return_value = alert_data
 
-        with patch("keep.providers.keep_provider.keep_provider.Pusher"), \
+        with patch("keep.providers.keep_provider.keep_provider.process_event"), \
              patch.object(provider, "logger"):
             # This should work: alert param provides the alert data
             result = provider._notify_alert(alert=alert_data)

